### ISSUE TITLE
Slow aray function used in loop warning dont show inside switch

### ISSUE THREAD

**Issue by JanGalek** [kalessil/phpinspectionsea#1498]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Plugin         | Php Inspections (EA Ultimate), 3.0.7 |
| Language level | PHP 5.5 |

Current behaviour
--
```
        foreach ($deliveryData as $key => $params) {
            switch ($key) {
                case "SB":
                    $postData = array_merge($postData, $this->getSmallBoxDelivery($params));
                    break;
                case "BB":
                    $postData = array_merge($postData, $this->getBigBoxDelivery($params));
                    break;
                case "SHD":
                    $postData = array_merge($postData, $this->getSHDDelivery($params));
                    break;
            }
        }
```
It doesn't show warning `slow array function used in loop`

Expected behaviour
--
Its should show warning `slow array function used in loop`

Environment details
---
PhpStorm 2019.3.2
Build #PS-193.6015.48, built on January 22, 2020
Licensed to Dixons Retail Plc (Hemel) / Jan Galek
Subscription is active until May 27, 2020
Runtime version: 11.0.5+10-b520.30 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Linux 5.4.13-050413-generic
GC: ParNew, ConcurrentMarkSweep
Memory: 1912M
Cores: 8
Registry: run.processes.with.pty=TRUE, bigger.font.in.project.view=true, ide.balloon.shadow.size=0
Non-Bundled Plugins: BashSupport, Key Promoter X, com.chrisrm.idea.MaterialThemeUI, com.kalessil.phpStorm.phpInspectionsUltimate, com.mnr.java.intellij.idea.plugin.base64helper, de.espend.idea.php.annotation, fr.adrienbrault.idea.symfony2plugin, mobi.hsz.idea.gitignore, org.psliwa.idea.composer, ru.adelf.idea.dotenv

**Comment by kalessil** [comment]
Oh, how I could miss that. Thank you for sharing @JanGalek!

**Comment by kalessil** [comment]
Fixed!

**Comment by JanGalek** [comment]
nice, thanks
